src/extension.js

```diff
@@ -16,7 +16,8 @@ const P_TITLE = 'Polacode 📸'
 function activate(context) {
   const htmlPath = path.resolve(context.extensionPath, 'webview/index.html')
 
-  let lastUsedImageUri = vscode.Uri.file(path.resolve(homedir(), 'Desktop/code.png'))
+  let lastUsedImageUri
+  let lastFile
```

Comment:
* Set lastUserImageUri to undefined until it is assigned for the first time during the call to the `shoot` event.
* Added variable to contain data on the last updated TextDocument object.